ubuntu-17.10-ppc64le-openstack.json

```diff
@@ -0,0 +1,73 @@
+{
+  "builders": [
+    {
+      "boot_command": [
+        "<wait>",
+        "c<enter>",
+        "linux /install/vmlinux",
+        " console-setup/ask_detect=false",
+        " console-setup/layoutcode=us",
+        " console-setup/modelcode=pc105",
+        " debconf/frontend=noninteractive",
+        " debian-installer=en_US.UTF-8",
+        " fb=false",
+        " kbd-chooser/method=us",
+        " keyboard-configuration/layout=USA",
+        " keyboard-configuration/variant=USA",
+        " locale=en_US.UTF-8",
+        " netcfg/get_hostname=unassigned-hostname",
+        " netcfg/get_domain=unassigned-domain",
+        " url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/ubuntu-17.10/preseed-ppc64.cfg<wait>",
+        " ---",
+        "<enter><wait>",
+        "initrd /install/initrd.gz",
+        "<enter><wait>",
+        "boot<enter><wait>"
+      ],
+      "accelerator": "kvm",
+      "boot_wait": "6s",
+      "disk_size": 4096,
+      "headless": true,
+      "vnc_bind_address": "0.0.0.0",
+      "http_directory": "http",
+      "iso_checksum": "da66eb3aeebaff2073b648c16a79dc9368fd11f36d049847943b6951524c47aa",
+      "iso_checksum_type": "sha256",
+      "iso_url": "{{user `mirror`}}/releases/17.10/release/ubuntu-17.10.1-server-ppc64el.iso",
```

Comment:
I don't think this file exists? When I look at the `sha256sums` file I can see that `ubuntu-17.10-server-ppc64el.iso` is the name file, just with a different name.